src/Tizen.NUI/src/public/Common/Vector3.cs

```diff
@@ -96,122 +99,50 @@ internal Vector3(Vector3ChangedCallback cb, float x, float y, float z) : this(In
         /// Actual value is (1.0f,1.0f,1.0f).
         /// </summary>
         /// <since_tizen> 3 </since_tizen>
-        public static Vector3 One
-        {
-            get
-            {
-                global::System.IntPtr cPtr = Interop.Vector3.OneGet();
-                Vector3 ret = (cPtr == global::System.IntPtr.Zero) ? null : new Vector3(cPtr, false);
-                if (NDalicPINVOKE.SWIGPendingException.Pending) throw new InvalidOperationException("FATAL: get Exception", NDalicPINVOKE.SWIGPendingException.Retrieve());
-                return ret;
-            }
-        }
+        public static Vector3 One => one;
 
         /// <summary>
         /// The vector representing the x-axis.
         /// </summary>
         /// <since_tizen> 3 </since_tizen>
-        public static Vector3 XAxis
-        {
-            get
-            {
-                global::System.IntPtr cPtr = Interop.Vector3.XaxisGet();
-                Vector3 ret = (cPtr == global::System.IntPtr.Zero) ? null : new Vector3(cPtr, false);
-                if (NDalicPINVOKE.SWIGPendingException.Pending) throw new InvalidOperationException("FATAL: get Exception", NDalicPINVOKE.SWIGPendingException.Retrieve());
-                return ret;
-            }
-        }
+        public static Vector3 XAxis => PositionAxis.X;
```

Comment:
```suggestion
        private static readonly xaxis = new Vector3(1.0f, 0.0f, 0.0f);
        public static Vector3 XAxis => xaxis;
```

Can we freely convert from `Vector3` to `Position`? I think we'd better newly define `xaxis` for `Vector3` class